draft-ietf-httpbis-compression-dictionary.md

```diff
@@ -326,14 +306,27 @@ Link: <https://example.org/dict.dat>; rel="compression-dictionary"
 
 # Dictionary-Compressed Brotli
 
-The "br-d" content encoding identifies a resource that is a
+The "dcb" content encoding identifies a resource that is a
 "Dictionary-Compressed Brotli" stream.
 
-A "Dictionary-Compressed Brotli" stream is a Brotli {{RFC7932}} stream for a
-response that has been compressed with an external dictionary using a
-compression window not larger than 16 MB.
+A "Dictionary-Compressed Brotli" stream is a binary stream that starts with a
+36-byte header and is followed by a Brotli {{RFC7932}} stream of the response
+that has been compressed with an external dictionary using a compression window
+not larger than 16 MB.
```

Comment:
I think that it's about this point that you might acknowledge that describing a thing completely in one sentence is hurting readability a bit.

```suggestion
A "Dictionary-Compressed Brotli" stream has a fixed header that is followed by
a Brotli {{RFC7932}} stream. The header consists of a fixed 4 byte sequence
and a 32 byte hash of the external dictionary that was used.  The Brotli stream
is created using the referenced external dictionary and a compression window
that is at most 16 MB in size.
```